tests/docker2earth/args-before-from.Dockerfile

```diff
@@ -4,7 +4,6 @@ ARG GO_MINOR=16
 ARG GO_VERSION="${GO_MAJOR}.${GO_MINOR}"
 FROM "${BASE}:${GO_VERSION}"
 WORKDIR /go/src/github.com/alexellis/href-counter/
-RUN go get -d -v golang.org/x/net/html
```

Comment:
docker2earth I've fixed in #32 by upgrading it to latest go and using a go.sum + go.mod file.